_articles/bluetooth.md

```diff
@@ -13,8 +13,68 @@ section: network-troubleshooting
 
 ---
 
+# Important Notes About Bluetooth
+
+Bluetooth is a bit odd. 
+There are a lot of factors that go into whether Bluetooth devices work as expected.
+
+
+## Bluetooth version:
+
+Bluetooth 5.0 is backwards compatible with older Bluetooth versions but, older bluetooth versioned devices are not always compatible with newer devices.
```

Comment:
Shouldn't the comma be before the conjunction `but`?

Reply:
Yes, you are correct. I clearly didn't proofread this enough as I was copying things out of my ticket notes. Lol. I was def planning on having people review this before merging. Thanks for the proofreading.